tests/src/test/scala/common/rest/WskRestOperations.scala

```diff
@@ -1295,6 +1323,11 @@ trait RunRestCmd extends Matchers with ScalaFutures with SwaggerValidator {
     Try(resp.entity.toStrict(timeout).map { _.data }.map(_.utf8String).futureValue).getOrElse("")
   }
 
+  def getReqData(req: HttpRequest): String = {
+    val timeout = 5.seconds
+    Try(req.entity.toStrict(timeout).map { _.data }.map(_.utf8String).futureValue).getOrElse("")
+  }
```

Comment:
The duplication here seems unnecessary. HttpResponse and HttpRequest have a common supertype, HttpMessage. Changing getResData to, for example, `entityToString` taking a HttpMessage might make sense.